main.go

```diff
@@ -131,8 +133,9 @@ func main() {
 		Client: mgr.GetClient(),
 	}
 
-	loggedcluster.O.LoggingEnabled = true
+	loggedcluster.O.EnableLoggingFlag = enableLogging
```

Comment:
I would keep loggedcluster.O.LoggingEnabled = enableLogging and have logging enabled kept as a property everywhere but that's nice :)